ui/src/components/Employee/Create/index.tsx

```diff
@@ -26,67 +30,105 @@ interface CreateEmployeeProps {
 const CreateEmployeeForm: React.FC<FormikProps<CreateEmployeeFormValues> & CreateEmployeeProps> = (
   p,
 ) => {
-  if (p.organizations.length && !p.values.organization_id) {
+  const [isModalOpen, setModalOpen] = useState<boolean>(false);
+  const [deleteTeam] = useAsyncThunk(deleteEmployeeAction, {
```

Comment:
Looks like a type here. It should be `deleteEmployee`.
```suggestion
  const [deleteEmployee] = useAsyncThunk(deleteEmployeeAction, {
```